src/codeu/chat/common/ConversationHeader.java

```diff
@@ -63,4 +63,21 @@ public ConversationHeader(Uuid id, Uuid owner, Time creation, String title) {
     this.title = title;
 
   }
+
+  @Override
+  public boolean equals(Object toCompare)
```

Comment:
If the ConversationHeader class and Messages class are ultimately using the equals() of Uuid and Time only, should the hashcode function still be overwritten? Or should I call the haschode functions of those?